packages/core/components/DirectoryTree/DirectoryTree.module.css

```diff
@@ -1,12 +1,13 @@
 .container {
-    --filter-display-height: 50px;
+    display: flex;
+    flex-direction: column;
     position: relative;
 }
 
 .filter-display-bar {
```

Comment:
Looks like there's something moderately funky happening: the bottom of the filter display bar is disappearing behind the search box UI:
<img width="715" alt="Screen Shot 2021-09-07 at 9 59 46 AM" src="https://user-images.githubusercontent.com/8576080/132383373-ed55647b-b1ee-415c-a518-7eb37f6f8564.png">

Reply:
Should be solved